include/deal.II/base/function_signed_distance.h

```diff
@@ -124,6 +126,51 @@ namespace Functions
       const Tensor<1, dim> normal;
     };
 
+
+    /**
+     * Signed-distance level set function to an ellipsoid defined by:
+     *
+     * @f[
+     * \sum_{i=1}^{dim} \frac{(x_i - c_i)^2}{R_i^2} = 1
+     * @f]
+     *
+     * Here, $c_i$ are the coordinates of the center of the ellipsoid and $R_i$
+     * are the elliptic radii. This function is zero on the ellipsoid, negative
+     * inside the ellipsoid and positive outside the ellipsoid.
+     *
+     * @ingroup functions
+     */
+    template <int dim>
+    class Ellipsoid : public Function<dim>
+    {
+    public:
+      /**
+       * Constructor, takes the center and radii of the ellipsoid.
+       */
+      Ellipsoid(const Point<dim> &             center    = Point<dim>(),
+                const std::array<double, dim> &radii     = {1., 1.},
+                const double                   tolerance = 1e-14,
+                const unsigned int             max_iter  = 10);
+
+      double
+      value(const Point<dim> & point,
+            const unsigned int component = 0) const override;
+
+    private:
+      double
+      evaluate_ellipsoid(const Point<dim> &point) const;
```

Comment:
Could you add the formula from above here so that one know what "evaluate" means.